src/cfn/loadStackArgs.ts

```diff
@@ -0,0 +1,167 @@
+import * as aws from 'aws-sdk';
+import * as fs from 'fs';
+import * as _ from 'lodash';
+import * as pathmod from 'path';
+import {GenericCLIArguments} from '../cli-util';
+import configureAWS from '../configureAWS';
+import getCurrentAWSRegion from '../getCurrentAWSRegion';
+import {logger} from '../logger';
+import {ExtendedCfnDoc, transform} from '../preprocess';
+import * as yaml from '../yaml';
+import {runCommandSet} from './runCommandSet';
+import {StackArgs} from './types';
+import {filter} from '../preprocess/filter';
+
+export function recursivelyMapValues<T extends object>(o: T, f: (a: any) => any): T {
+  return _.mapValues(o, function(a: any) {
+    if (typeof a === 'object') {
+      return recursivelyMapValues(a, f);
+    } else {
+      return f(a);
+    }
+  }) as T;
+}
+
+export async function addDefaultNotificationArn(args: StackArgs): Promise<StackArgs> {
```

Comment:
This isn't used outside of the module.

```suggestion
async function addDefaultNotificationArn(args: StackArgs): Promise<StackArgs> {
```